dll/directx/wine/d3dx9_36/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 
-add_definitions(-D__WINESRC__ -Dcopysignf=_copysignf)
+add_definitions(-D__WINESRC__ -Dcopysignf=_copysignf -D_D3DX9_VER=36)
```

Comment:
This is a little bit scary. Does that mean Wine can have different code between the different version, but we'll just keep forwarding?

Reply:
Yeap, I'm afraid eventually we'll have to compile the code over and over for each module like upstream when the bugs appear and become a priority.

Reply:
It is currently only used to extend a vTable. Using maximum is fine.